packages/muelu/src/MueCentral/MueLu_FactoryManager_def.hpp

```diff
@@ -143,7 +136,7 @@ namespace MueLu {
         // GetFactory("Ptent"): we need to use the same factory instance for both "P" and "Nullspace"
         RCP<Factory> factory;
 #ifdef HAVE_MUELU_KOKKOS_REFACTOR
-        if (useKokkos)
+        if (useKokkos_)
           factory = rcp(new SaPFactory_kokkos());
```

Comment:
Why can't this use the macro `MUELU_KOKKOS_FACTORY` as is done for TentativePFactory above?

Reply:
The macro returns a `const FactoryBase` which does not have the method `SetFactory` that the derived `Factory` class has. We could use the macro and cast back to `Factory`. Would you prefer that?

Reply:
That makes sense.  I don't see a need to change from what you're doing now.